addons/optionsmenu/config.cpp

```diff
@@ -30,7 +30,10 @@ class CfgAddons {
 #include "gui\pauseMenu.hpp"
 
 class ACE_Extensions {
-    extensions[] += {"ace_clipboard"};
+    class ace_clipboard {
+        windows = 1;
+        client = 1;
```

Comment:
Is this correct? Could this be used on server theoretically?

Reply:
yes. you could copy to the servers clipboard if it were running on a windows server.
But I'd see that more as a security issue than a feature.

Reply:
How is that more of a security issue than on client?